### DISCUSSION TITLE
Taiko: Add a setting that makes notes go into the health bar, like in stable, rather than floating off the left side of the screen

### DISCUSSION THREAD

**Discussion by OpenSauce04** [ppy/osu#22023] [Ideas] [answered]
Over time I've checked out Lazer many times, and every time the way the notes float off the left side of the screen rather than how it looks in stable has always been jarring to me
Some kind of option for this would be nice

**Comment by OpenSauce04** [comment]


**Reply by mk56-spn** [reply]
Please do not do bumps. Devs will get to this when they can.

**Reply by OpenSauce04** [reply]
Alright

**Comment by peppy** [comment, marked as answer]
At very least this should be the case for legacy skins. But I'm not sure how well this will work when the health bar can be moved anywhere.

**Reply by OpenSauce04** [reply]
In retrospect, where I thought this was what felt jarring about Lazer taiko, what I was actually feeling was more than likely what was addressed in #22142

Still, probably a good idea to add this to legacy skins

**Reply by unknowwiiplayer** [reply]
Maybe the solution is to add some skinning parameters like target (where the notes should land), inital vector (this allows for TnT 2P like skins) and target gravity (to adjust the trajectory of the note).
